ElectronicObserver/Extensions.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.IO;
+using System.Windows;
+using System.Windows.Media.Imaging;
+using Avalonia.Media.Imaging;
+using CroppedBitmap = System.Windows.Media.Imaging.CroppedBitmap;
+
+namespace ElectronicObserver;
+
+public static class Extensions
+{
+	public static BitmapImage ToImageSource(this Bitmap bitmap)
+	{
+		using MemoryStream memoryStream = new();
+		bitmap.Save(memoryStream);
+		memoryStream.Seek(0, SeekOrigin.Begin);
+
+		BitmapImage bitmapImage = new();
+		bitmapImage.BeginInit();
+		bitmapImage.CacheOption = BitmapCacheOption.OnLoad;
+		bitmapImage.StreamSource = memoryStream;
+		bitmapImage.EndInit();
+
+		return bitmapImage;
+	}
+
+	public static CroppedBitmap ToCroppedImageSource(this Bitmap bitmap)
```

Comment:
Since offset seems hardcoded, maybe should rename this method to ToCroppedShipImageSource ?

Reply:
Changed the name and added a summary to make it clear.